### DISCUSSION TITLE
Activity status empty

### DISCUSSION THREAD

**Discussion by vilewired** [Readme-Workflows/recent-activity#284] [Q&A] [answered]
I update my readme with a few github actions, some run more often, some run less often. Due to that, I have a lot of commits in my README file. I already added my personal repo to the ignored repos because my entire list would only contain pushes to that repo. Now I think what happens is, that it looks at my past 20 activities (which are all the ignored repo) so it updates my activity to be empty instead of just either ignoring it completely, or fetching the last 20 activities which aren't in that repo. I can change one of my other actions so it doesn't push the commit as me. I don't know for sure if that's the case, but if it is then it would be awesome to make this a new feature / including it in the next update as a fix.

**Comment by abhijoshi2k** [comment, marked as answer]
I think changing commit email to GitHub bot might fix your issue. Generally 100 latest commits are fetched. If all are from ignored repos, it'll not show any activity.

**Reply by vilewired** [reply]
I'll give it a try :)

**Reply by vilewired** [reply]
I changed all my actions to commit as the github Bot and it seems to work so far :) 
I think you could fix it code-wise too by looping over the commits until you have reached the amount of commits you want to display, but I don't know if github limits the amount of commits you can fetch. Either way, this is a very nice project, keep it up! :+1:

**Reply by abhijoshi2k** [reply]
It's GitHub that limits to 100 recent activities. So we cannot get more events.

**Reply by vilewired** [reply]
I see, alright thank you :)
